.github/workflows/slow-tests.yml

```diff
@@ -74,24 +72,35 @@ jobs:
       - name: Pip install
         run: |
           source activate trl
-          pip install -e . --no-deps
-          pip install pytest-reportlog
-      
-      - name: Run common tests on single GPU
```

Comment:
I propose to support common tests in a follow up PR, right now many of them fail because of device mismatch problems, I think it is ok to not have them for now as these tests are run anyway on the CI